Views/PricePartSettings.Edit.cshtml

```diff
@@ -0,0 +1,51 @@
+@model PricePartSettingsViewModel
+
+<fieldset class="form-group" asp-validation-class-for="PricePartSettings">
+    <label asp-for="CurrencySelectionMode">@T["Currency selection mode"]</label>
+
+    <div class="input-group">
+        <div class="input-group-append">
+            <select asp-for="CurrencySelectionMode"
+                    asp-items="@(new SelectList(Model.CurrencySelectionModes, "Text", "Value"))"></select>
+        </div>
+    </div>
+
+    <span class="hint">@T["Specifies the currency configuration for this PricePart."]</span>
+
+    <fieldset class="input-group" id="speccurr">
+        <div class="input-group">
+            <div class="input-group-append">
+                <select asp-for="CurrencyIsoCode"
+                        asp-items="@(new SelectList(Model.Currencies, "Text", "Value"))"></select>
+            </div>
+        </div>
+
+        <span class="hint">@T["Currency used for specific currency selection mode."]</span>
+    </fieldset>
+</fieldset>
+
+<script at="Foot">
+$(function () {
+    $('#@Html.IdFor(m => m.CurrencySelectionMode)').change(function () {
+        var value = $('#@Html.IdFor(m => m.CurrencySelectionMode)').val();
+
+        if (value == "SpecificCurrency") {
```

Comment:
Can we use a nameof or something similar so this is more resilient to refactorings?

Reply:
Changed the viewmodel to include the editor type.